test/conftest.py

```diff
@@ -130,6 +130,12 @@ def db():
         )
         if db.pgver < "13":
             fp.write("wal_keep_segments = 100\n")
+
+        if LooseVersion(db.pgver) < LooseVersion("9.4"):
+            fp.write("wal_level = archive\n")
+        else:
+            fp.write("wal_level = logical\n")
```

Comment:
`logical` was introduced in 9.4

Reply:
We're not even testing below 9.6, I'd rather not introduce untested/dead code.

Reply:
True, but `PG_VERSIONS` (`conftest.py`) still lists versions back to 9.2, which is why I tried to be compatible with that change. I'm going to remove that conditional, as well as versions older than 9.6 from the mentioned list.